dev-support/benchmark/grouping/grouping_conf

```diff
@@ -0,0 +1,16 @@
+#Grouping Benchmark
```

Comment:
as this is kind of documentation on how to configure the runner, please also add some documentation to the single lines